packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/process-node.js

```diff
@@ -414,26 +414,6 @@ const copyFileToStaticAndReturnUrlPath = async (fileNode, helpers) => {
   return relativeUrl
 }
 
-const filterMatches =
```

Comment:
This function isn't being used anywhere 🤔 not sure why but it was dead code so I deleted it